CONTRIBUTING.md

```diff
@@ -243,7 +243,7 @@ For a bugfix, the end of the PR description (that is, the first comment on the P
 
 NOTE: it's best not to add the issue reference to your commit message, as github will pollute the conversation on the ticket with notifications every time you commit.
 
-If a commit purely refactors and is not intended to change behaviour,
+If a commit purely refactors and is not intended to change behavior,
```

Comment:
I'm American, so I won't complain, but I'm pretty sure this is the reason we seceded from the UK.